samples/showcase/cinema/sample.md

```diff
@@ -0,0 +1,65 @@
+---
+title: Cinema Showcase
+summary: Implements a fictional cinema showing a saga that aggregates ticket sales.
+reviewed: 2023-09-13
+component: Core
+---
+
+
+## Scenario
+
+Keep track of ticket sales at cinemas and feature the film with the most tickets sold for the current month.
```

Comment:
A couple of sentences on why this showcase sample has been created (even though it may be mentioned in the accompanying video).

Reply:
We will also be adding a link to the video once it's available